### PR TITLE
[DOC-3586] NuoDB Insights: Setup manually Docker

### PR DESCRIPTION
**Pull request by rkampurath**
Added ordered list edited text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,8 +122,12 @@ docker run -d --name influxdb \
       -v $PWD/deploy/initdb.sh:/docker-entrypoint-initdb.d/initdb.sh \
       influxdb:1.8
 ```
+      
+      
+   NOTE: NuoDB Collector only supports InfluxDB version 1.X.
```
Comment (jleslie85) [inline README.md:127]: nit: this sentence may format better if lined up with other sub statements like, "Use ..." and "All NuoDB components..." above.

File: README.md [hunk 2]
```diff
@@ -128,10 +137,12 @@ docker run -d --name grafana \
       grafana/grafana:7.5.4
 ```
 
-You can now start your NuoDB database with the NuoDB Collector.
-For more info on how to start your database, please refer to the [NuoDB Docker Blog Part I](https://nuodb.com/blog/deploy-nuodb-database-docker-containers-part-i) and the readme for [NuoDB Collector](https://github.com/nuodb/nuodb-collector).
+4. If you haven't already, [start your NuoDB database](https://nuodb.com/blog/deploy-nuodb-database-docker-containers-part-i).
+      
+5. Start the [NuoDB Collector](https://github.com/nuodb/nuodb-collector) daemons for each NuoDB database process.      
+
+   Use the example NuoDB YCSB workload generator given below to explore the various dashboards.
```
Comment (jleslie85) [inline README.md:144]: This looks like a 6th step. Maybe phase as, 

6. "Start the NuoDB YCSB workload generator to explore the various dashboards with live monitoring data."

File: README.md [hunk 3]
```diff
@@ -128,10 +137,12 @@ docker run -d --name grafana \
       grafana/grafana:7.5.4
 ```
 
-You can now start your NuoDB database with the NuoDB Collector.
-For more info on how to start your database, please refer to the [NuoDB Docker Blog Part I](https://nuodb.com/blog/deploy-nuodb-database-docker-containers-part-i) and the readme for [NuoDB Collector](https://github.com/nuodb/nuodb-collector).
+4. If you haven't already, [start your NuoDB database](https://nuodb.com/blog/deploy-nuodb-database-docker-containers-part-i).
```
Comment (sivanov-nuodb) [inline README.md:140]: Isn't it better to point to our official doc - https://doc.nuodb.com/nuodb/latest/quick-start-guide/docker/?

Comment (rkampurath) [reply README.md:140]: Thank you for your suggestion. I will replace the link.

**Pull request merged**